dashboard/v1.1/src/pages/Monitoring/Matrix.tsx

```diff
@@ -256,6 +272,11 @@ const Matrix: FC<MatrixProps> = ({
   };
   return (
     <>
+      <SearchBar
```

Comment:
Please add proper styling to your components before you request a review. After you have done that, you can re-request a review in the PR settings(if you have access to).